docs/pages/includes/cloud/cloudmanagedadvisory.mdx

```diff
@@ -0,0 +1,7 @@
+<Details scopeOnly={true} scope={["cloud"]} title="Teleport Cloud">
+This guide deploys the Teleport Auth Service and Proxy Service, which are fully managed in Teleport Cloud. If you are a Teleport Cloud customer, we recommend following our guides for deploying specific services on your Teleport Nodes. Here are some examples:
+
+- [Application Access](../application-access/getting-started.mdx)
+- [Database Access](../database-access/getting-started.mdx)
+- [Server Access](../server-access/getting-started.mdx)
```

Comment:
Should we go ahead and list Kubernetes Access and Desktop Access here? Then we can remove "Here are some examples" above and replace the final "." in the preceding paragraph with a ":".

Reply:
done